site/viamguides/src/yolo-training/yolo-training.md

```diff
@@ -0,0 +1,307 @@
+1. Click **Save** in the top right to save and apply your configuration changes.
+
+Your YOLO-ONNX ML model service is now configured with your custom model. Next, let's set up a vision service to use this model for object detection.
+
+<!-- ------------------------ -->
+## Configure the Vision Service
+Duration: 5
+
+1. In [the Viam app](https://app.viam.com/fleet/locations), find the **CONFIGURE** tab.
+1. Click the **+** icon in the left-hand menu and select **Service**.
+1. Select `vision`, and find the `ML Model` module. Give your vision service a descriptive name, such as `yolo-vision`. Click **Create**.
+   <img src="assets/add-vision-service.png" alt="add vision service" width="450" />
+1. In the **Configure** panel of your vision service, set the **ML Model** to your ML Model service (`mlmodel-1`).
+   <img src="assets/configure-vision-ml-model.png" alt="set vision service ml model" width="450" />
+1. Move the **Minimum confidence threshold** slider to `0.4`. This sets the minimum confidence level for object detections to be considered valid.
+   <img src="assets/configure-vision-confidence.png" alt="set vision service confidence" width="450" />
+1. Find and select your camera component in the **Depends on** section (`camera-1`).
+   <img src="assets/vision-depends-on.png" alt="set dependent camera" width="450" />
+1. Click **Save** in the top right to save and apply your configuration changes. This might take a few moments.
+1. Expand the **TEST** panel of your vision service. You'll see a live feed of your configured webcam and a section **Labels**.
+   Test out your model! Try showing your objects to your webcam. When detected, an item will appear with the label the vision service thinks it is seeing and its confidence level.
+   You can also try showing multiple objects and see how well your model detects them!
+   ![computer vision powered machine; gif showing an object being shown to the camera with a bounding box around it and a confidence level for its detection](assets/yolo-vision-demo.gif)
+
+   > aside negative
+   > **Troubleshooting**: Having trouble detecting the correct object (or anything at all)? Try setting the **Minimum confidence threshold** to something smaller, save your changes, and see if that helps. Additionally, make sure you have enough light for your webcam to detect clearer images, keep your background and surroundings free of potential visual clutter, and hold up your object for at least a few seconds (so that the vision service can get a clear image to interpret). If this all still doesn't work, you many need to add more images or better quality images to your model. Luckily, you can repeat the steps in **Create a dataset and capture training data**, train a new version of your model, then try using it in your ML model service. The more data the better, so at the very least, more images of your objects _should_ make your model more accurate.
```

Comment:
This section might be more legible if it was bulleted. And it's probably an important section for a lot of users.

Also typo "you many need"